### ISSUE TITLE
A11y: Apply filter button behavior for screenreaders

### ISSUE THREAD

**Issue by angelathe** [CDCgov/dibbs-ecr-viewer#1148]
## What needs to be done

Currently, the Apply Filter button is disabled until a filter has been changed. This means screen reader users cannot navigate to or hear the button when it’s disabled (or know when it gets enabled), can be confusing.

Add a way to make sure it is known to screen readers that the button exists when disabled, and announce when the button becomes actionable (when the filters change).

## Why it needs to be done

This follows the accessibility audit scans of the eCR Viewer.

## Technical Notes

Maybe an aria alert for when the button is enabled? And/or helper text that mentions the filter button will enable after you've made your changes?

## Additional context

https://www.w3.org/WAI/ARIA/apg/patterns/
